plugins/com.google.gcp.eclipse.appengine.localserver/gcloud_serverdef.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tns:ServerRuntime
+   xmlns:tns="http://eclipse.org/jst/server/generic/ServerTypeDefinition"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://eclipse.org/jst/server/generic/ServerTypeDefinition ServerTypeDefinitionSchema.xsd"
+   name="GCloud" version="v94.0.0">
+   
+<property id="cloudSdkDirectory"
+  label="Directory where gcloud is installed"
+  type="directory"
+  context="runtime"
+  default="/usr/local/google/home/elharo/google-cloud-sdk" />
```

Comment:
I don't want to leave this empty, since that makes it less clear what's required here. Any suggestions for a reasonable default? I wonder if we can somehow link to the user's home directory.